Resources/Prototypes/Entities/Clothing/Back/smuggler.yml

```diff
@@ -0,0 +1,1009 @@
+        prob: 0.02
+        orGroup: Keys
+      - id: EncryptionKeyCargo
+        prob: 0.02
+        orGroup: Keys
+      - id: EncryptionKeyService
+        prob: 0.02
+        orGroup: Keys
+      - id: EncryptionKeyRobo
+        prob: 0.02
+        orGroup: Keys
+      - id: EncryptionKeyScience
+        prob: 0.02
+        orGroup: Keys
+      - id: EncryptionKeyMedical
+        prob: 0.02
+        orGroup: Keys
+      - id: EncryptionKeyEngineering
+        prob: 0.02
+        orGroup: Keys
+      - id: EncryptionKeySyndie
+        prob: 0.02
+        orGroup: Keys
+      - id: EncryptionKeySecurity
+        prob: 0.02
+        orGroup: Keys
+      - id: EncryptionKeyCommand
+        prob: 0.02
+        orGroup: Keys
+      - id: EncryptionKeyStationMaster
+        prob: 0.001
+        orGroup: Keys
+      - id: SpaceCash
+        prob: 0.02
+        orGroup: Keys  #1x small
+
+- type: entity
+  parent: ClothingBackpackSatchelSmuggler
+  id: ClothingBackpackSatchelSmugglerFour
+  noSpawn: true
+  components:
+  - type: Sprite
+    sprite: Clothing/Back/Satchels/smuggler.rsi
+    state: icon
+  - type: StorageFill
+    contents:
+
+      - id: SpaceCash100
+        prob: 0.90
+        orGroup: Materials
+      - id: MaterialDiamond1
+        prob: 0.15
+        maxAmount: 10
+        orGroup: Materials
+      - id: MaterialBananium1
+        prob: 0.15
+        maxAmount: 10
+        orGroup: Materials
+      - id: IngotGold1
+        prob: 0.15
+        maxAmount: 10
+        orGroup: Materials
+      - id: IngotSilver1
+        prob: 0.15
+        maxAmount: 10
+        orGroup: Materials
+      - id: SheetPlasma1
+        prob: 0.15
+        maxAmount: 10
+        orGroup: Materials
+      - id: SheetUranium1
+        prob: 0.15
+        maxAmount: 10
+        orGroup: Materials
+      - id: SpaceCash
+        prob: 0.15
+        orGroup: Materials #1x medium
+
+      - id: SpaceCash100
+        prob: 0.98
+        orGroup: Implanters
+      - id: LightImplanter
+        prob: 0.025
+        orGroup: Implanters
+      - id: BikeHornImplanter
+        prob: 0.025
+        orGroup: Implanters
+      - id: SadTromboneImplanter
+        prob: 0.025
+        orGroup: Implanters
+      - id: StorageImplanter
+        prob: 0.025
+        orGroup: Implanters
+      - id: SpaceCash
+        prob: 0.025
+        orGroup: Implanters #1x small
+
+      - id: SpaceCash100
+        prob: 0.95
+        orGroup: Cells
+      - id: PowerCellHyper
+        prob: 0.05
+        orGroup: Cells
+      - id: PowerCellMicroreactor
+        prob: 0.05
+        orGroup: Cells
+      - id: PowerCellHigh
+        prob: 0.05
+        orGroup: Cells
+      - id: SpaceCash
+        prob: 0.05
+        orGroup: Cells #1x small
+
+      - id: SpaceCash100
+        prob: 0.99
+        orGroup: Syndicate
+      - id: Telecrystal1
+        prob: 0.05
+        maxAmount: 10
+        orGroup: Syndicate
+      - id: Emag
+        prob: 0.05
+        orGroup: Syndicate
```

Comment:
While this does have some uses that aren't terrible, I also don't know if its a good idea to give to non-antags.